samples/sid_end_device/sysbuild/mcuboot/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	chosen {
+		zephyr,code-partition = &boot_partition;
```

Comment:
@nordicjm I move this change form your fixup commit https://github.com/ktaborowski/sdk-sidewalk/pull/10/commits/36dde823847f56c0875704bde7f47e62651cd94c
Could you explain what it is for?